### PR TITLE
Add km test

### PR DESCRIPTION
**Pull request by kning**
Adds mock data test of kaplan meier model, similar to what we have internally.

### CODE REVIEW

**Comment by kning** [comment]
@erikbern review please. one of the hound violations i'd like to leave if cool, it's just kind of a nasty one-liner list comprehension :/.

**Comment by kning** [comment]
also not sure why this build failed...

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+autograd
```
[the remark this answers is not in the document]
Comment (erikbern) [reply requirements.txt]: i mean requirements.txt

**Comment by erikbern** [comment]
It's failing on something with pytest (can you see the travis build output? not sure if you need to be authenticated)

**Comment by kning** [comment]
Error in the build (only for py 2.7 and py3.6, passes for py3.5 strangely enough):

`The command "py.test -s -v --cov=convoys ." exited with 1.`

Tests pass fine locally though.

File: requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (kning) [reply requirements.txt]: couldn't get `setup.py install` to work locally, so thought this was nice to have as an alternative solution.

**Comment by erikbern** [comment]
Thanks @kning !

**Pull request merged**